<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="number1_label">
       <property name="text">
        <string>Number1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="number1_doubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="number2_label">
       <property name="text">
        <string>Number2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="number2_doubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="multiply_pushButton">
       <property name="text">
        <string>Multiply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="divide_pushButto">
       <property name="text">
        <string>Divide</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string>Result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="result_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="sendSignalToServer_pushButton">
     <property name="text">
      <string>Send Signal to Server</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
